BBBadgeBarButtonItem/BBBadgeBarButtonItem.m

```diff
@@ -142,6 +141,7 @@ - (void)setBadgeValue:(NSString *)badgeValue
         self.badge.backgroundColor      = self.badgeBGColor;
         self.badge.font                 = self.badgeFont;
         self.badge.textAlignment        = NSTextAlignmentCenter;
+        self.badge.layer.masksToBounds  = YES;
 
```

Comment:
Thanks, that's better but I guess I will have to revisit the podfile in the future to remove that hard coded commit